content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,333 @@ Download the Istio release which includes installation files, samples, and the
+{{< /text >}}
+
 {{< /warning >}}
 
+**Follow these instructions if your environment does not have an external load balancer and choose a node port instead.**
+
+Set the ingress ports:
+
+{{< text bash >}}
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+$ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
+{{< /text >}}
+
+_GKE:_
+
 {{< text bash >}}
-$ kubectl label namespace <namespace> istio-injection=enabled
+$ export INGRESS_HOST=<workerNodeAddress>
 {{< /text >}}
 
-Alternatively, you can manually inject Envoy containers in your application pods before deploying them, using
-[`istioctl kube-inject`](/docs/reference/commands/istioctl/#istioctl-kube-inject):
+You need to create firewall rules to allow the TCP traffic to the `ingressgateway` service's ports.
+Run the following commands to allow the traffic for the HTTP port, the secure port (HTTPS) or both:
 
 {{< text bash >}}
-$ istioctl kube-inject -f <your-app-spec>.yaml | kubectl apply -f -
+$ gcloud compute firewall-rules create allow-gateway-http --allow tcp:$INGRESS_PORT
+$ gcloud compute firewall-rules create allow-gateway-https --allow tcp:$SECURE_INGRESS_PORT
 {{< /text >}}
 
-## Deploy an application or the Bookinfo sample {#Bookinfo}
+_Docker For Desktop:_
 
-If you have an application ready to go, deploy it:
+{{< text bash >}}
+$ export INGRESS_HOST=127.0.0.1
+{{< /text >}}
+
+_Other environments (e.g., IBM Cloud Private, etc.):_
 
 {{< text bash >}}
-$ kubectl create -n <namespace> -f <your-app-spec>.yaml
+$ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
 {{< /text >}}
 
-Alternatively,
-[deploy the Bookinfo sample](/docs/examples/bookinfo/)
-to evaluate Istio's features for traffic routing,
-fault injection, rate limiting, etc. Then, explore the various
-[Istio tasks](/docs/tasks/) that interest you.
+{{< /tab >}}
+
+{{< /tabset >}}
+
+1.  Set `GATEWAY_URL`:
+
+    {{< text bash >}}
+    $ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+    {{< /text >}}
+
+1.  Ensure an IP address and port were successfully assigned to the environment variable:
+
+    {{< text bash >}}
+    $ echo $GATEWAY_URL
+    192.168.99.100:32194
+    {{< /text >}}
+
+### Verify external access {#confirm}
 
-## Next evaluation steps
+Confirm that the Bookinfo application is accessible from outside the cluster by
+navigating in your browser to `http://<your $GATEWAY_URL value>/productpage`.
 
-After installing Istio and deploying an application,
-see the following topics to explore more Istio features:
+## View the dashboard {#dashboard}
+
+Istio has several optional dashboards installed by the `demo` installation. The
```

Comment:
Is this saying the dashboards are installed by the installation?  I 'm not sure that's 100% accurate. On the flip side, if I install Kiali outside of Istio, can I still get at it's dashboard with the CLI? @esnible, do you know?

Reply:
The demo install definitely adds the Kiali dashboard per my testing. The others I'm not sure.